posit_workbench_acceptable_use_policy.md

```diff
@@ -0,0 +1,82 @@
+# Posit Workbench Acceptable Use Policy
+
+Original author(s): Terry McLaughlin
+
+Last updated: January 2023
+
+## Introduction
+
+
+
+### Purpose
+
+
+### Scope
+
+
+## Accessing Posit Workbench
+
+### Authorisation
+
+Line managers of PHS staff can authorise their requests for access to Posit Workbench.
+
+Non-PHS users must seek authorisation from ...
+
+### Changing User Access
+
+Changes to users' access can be requested by ...
+
+### User Access Tracking
+
+Public Health Scotland have a fixed number of named-user licenses for Posit Workbench, and therefore users accessing Posit Workbench will be tracked for the sole purpose of ensuring compliance with the license agreement.
+
+## Appropriate Usage
+
+### Cloud Services
+
+Posit Workbench is hosted in Microsoft Azure, a cloud computing platform.  Microsoft Azure is a pay-as-you-go platform, and therefore most activities incur cost to Public Health Scotland, particularly running a Workbench session (even when it is idle) and any data being saved out of the Microsoft Azure environment e.g. saving to a file share on the Stats server.
+
+Users must take steps to minimise these costs by ensuring that they only request the computing resources they need to carry out their work, and ensuring that their analyses are as efficient as possible at all times.
```

Comment:
Should we link to the guidance on choosing a session size here? Or is there other guidance about writing efficient code that would be more helpful?

https://github.com/Public-Health-Scotland/R-Resources/blob/terrymclaughlin_posit_workbench_and_kubernetes/posit_workbench_and_kubernetes.md

Reply:
Done.